src/inverted-index.js

```diff
@@ -0,0 +1,140 @@
+const utils = require('./utils.js');
+
+/**
+ * A class for Creating and searching an inverted index
+ */
+class Index {
+
+  /**
+   * [constructor Creates Object variables to hold
+   * 1. [jsonDatabase] Uploaded files
+   * 2. [indexFile] the indexed filenames and their contents
+   * 3. [searchResult] the search results
+   */
+  constructor() {
+    this.jsonDatabase = {};
+    this.indexFile = {};
+    this.searchResult = {};
```

Comment:
NITPICK
I'm not entirely sure you need searchResult. Instance variables are supposed to define the properties of an object. I'm not sure how searchResult is a property of index.